.teamcity/configurations/bazelBsp/bazelE2eTests.kts

```diff
@@ -60,6 +60,10 @@ object KotlinProjectTest : BazelBspE2ETestsBuildType(
     targets = "//e2e:kotlin_project_test",
 )
 
+object ScalaProjectTest : BazelBspE2ETestsBuildType(
```

Comment:
It seems this hasn't really added the test? Anyone has any idea what is going on?

Also how to debug the tests? Any tips?

Reply:
regarding debugging (if it doesn't work) @abrams27 may be of help

Reply:
> yes, it will require adding test in on other place - i'll make a commit with required change to your branch tommorow.

Thanks!